extension/src/paymentHandler/payment-handler.js

```diff
@@ -73,33 +62,18 @@ function _getPaymentHandlers(paymentObject) {
   if (!paymentObject.custom) return []
 
   const handlers = []
+
   if (
-    paymentObject.custom.fields.getPaymentMethodsRequest &&
-    !paymentObject.custom.fields.getPaymentMethodsResponse
-  )
-    handlers.push(getPaymentMethodsHandler)
+    paymentObject.custom.fields.createSessionRequest &&
+    !paymentObject.custom.fields.createSessionResponse
+  ) {
+    handlers.push(sessionRequestHandler)
+  }
   if (
     paymentObject.custom.fields.getCarbonOffsetCostsRequest &&
     !paymentObject.custom.fields.getCarbonOffsetCostsResponse
   )
     handlers.push(getCarbonOffsetCostsHandler)
-  if (
-    paymentObject.custom.fields.makePaymentRequest &&
-    !paymentObject.custom.fields.makePaymentResponse
-  ) {
-    const makePaymentRequestObj = JSON.parse(
-      paymentObject.custom.fields.makePaymentRequest
-    )
-    if (_requiresLineItems(makePaymentRequestObj))
```

Comment:
line items support is still required.

Reply:
https://jira.commercetools.com/browse/CTPI-761

Reply:
@praveenkumarct If we follow up this part as next step, I would suggest to keep this part this moment, so that it helps us to understand how to migrate line-item features for web component v5.0

In addition, please keep `multitenancy.spec.js` and `payment-handler-authorization.spec.js`. They are required all the time no matter which web component version we are using.

Reply:
Same here.